stylesheet.css

```diff
@@ -0,0 +1,275 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    margin: 0;
+    font-family: 'Poppins', sans-serif;
+    line-height: 1.6;
+}
+html{
+    scroll-behavior: smooth;
+}
+/* Nav bar styling */
+
+nav{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    background-color: #d1e0ef;
+    text-transform: uppercase;
+}
+.topnav{
+    padding: 12px;
+}
+nav a{
+    margin: 10px;
+}
+nav a:hover{
+    background-color:#fae3e5;
+    border-radius: 25px;
+    padding: 12px;
+}
+a:hover{
+    color: #e2431b;
+}
+/* Header styling  */
+header{
+    background-image: url("Images/Background.jpg");
+    height: 750px;
+    text-align: center;
+}
+.intro{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;    
+}
+.intro_item{
+    margin: 30px;
+    align-self: center;
+}
+.intro_item.intro_text{
+    background-color: #FFFFFF ;
+    padding: 100px 40px;
+    margin-left: 150px;
+    text-align: left;
+}
+.profile{
+    object-fit: cover;
+    border-radius: 50%;
+    height: 300px;
+    width: 300px;
+    display: block;
+    margin: auto;
+}
+p{
+    padding-bottom:10px;
+}
+/* header button styling */
+.header_button{
```

Comment:
I think you could make your buttons a bit dryer, if you had a 'base' button class that contains all the rules that are the same for all of your buttons, you could then create some shared class selectors to modify things like the colour and text sizes. Should help reduce some repetition

Reply:
Ah great thank you, that's worked really well.